src/fsharp/FSharp.Build-proto/FSharp.Build-proto.fsproj

```diff
@@ -47,11 +47,11 @@
     <Reference Include="$(FSCoreLKGPath)" />
     <Reference Include="System" />
     <Reference Include="System.Numerics" />
-    <Reference Include="Microsoft.Build.Framework, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
-    <Reference Include="Microsoft.Build.Engine, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
-    <Reference Include="Microsoft.Build, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
-    <Reference Include="Microsoft.Build.Utilities.v12.0" />
-    <Reference Include="Microsoft.Build.Tasks.v12.0" />
+    <Reference Include="Microsoft.Build.Framework, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
```

Comment:
Why is proto using VS 2010-era msbuild version, but everything else is using VS 2015 version?